### ISSUE TITLE
Add support for Gradient Boosting from scikit-learn

### ISSUE THREAD

**Issue by abhishek-finstack** [BayesWitnesses/m2cgen#232]
Is there any way to transpile sklearn GBM models? I know it's not supported in the library. But maybe some way to convert it into one of the supported models, and then transpile. Any leads would be appreciated. Thanks!

**Comment by StrikerRUS** [comment]
Hi @abhishek-finstack !

I believe you can transpile all underlying weak estimators stored in `estimators_` attribute with the help of `m2cgen`.

> estimators_  ndarray of DecisionTreeRegressor of shape (n_estimators, 1)
  https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.GradientBoostingRegressor.html

And then manually write a sumup function according to the following original code.

https://github.com/scikit-learn/scikit-learn/blob/fd237278e895b42abe8d8d09105cbb82dc2cbba7/sklearn/ensemble/_gb.py#L1610-L1611

https://github.com/scikit-learn/scikit-learn/blob/fd237278e895b42abe8d8d09105cbb82dc2cbba7/sklearn/ensemble/_gb.py#L608-L613

https://github.com/scikit-learn/scikit-learn/blob/fd237278e895b42abe8d8d09105cbb82dc2cbba7/sklearn/ensemble/_gb.py#L619-L620

https://github.com/scikit-learn/scikit-learn/blob/fd237278e895b42abe8d8d09105cbb82dc2cbba7/sklearn/ensemble/_gradient_boosting.pyx#L215-L227

**Comment by StrikerRUS** [comment]
I believe implementing support for `HistGradientBoostingRegressor`/`HistGradientBoostingClassifier` (#381) has higher priority.

**Comment by StrikerRUS** [comment]
Added into feature-request hub #491.
